hail/python/hailtop/config/deploy_config.py

```diff
@@ -128,28 +126,6 @@ async def get_metrics(request):  # pylint: disable=unused-argument,unused-variab
         log.info(f'serving paths at {base_path}')
         return root_app
 
-    async def addresses(self, service: str) -> List[Tuple[str, int]]:
```

Comment:
Was this dead code?

Reply:
Yes, it makes a request to the `address` service which no longer exists.